iambic/plugins/v0_1_0/github/github.py

```diff
@@ -825,25 +834,102 @@ def handle_detect_changes_from_eventbridge(
     repo_name = context["repository"]
     templates_repo = github_client.get_repo(repo_name)
     default_branch = get_remote_default_branch(templates_repo)
-    _handle_detect_changes_from_eventbridge(repo_url, default_branch)
+
+    _handle_detect_changes_from_eventbridge(
+        repo_url,
+        default_branch,
+        github_client=github_client,
+        templates_repo=templates_repo,
+    )
+
+
+def __get_config() -> Config:
```

Comment:
I think we're calling load_config multiple times in this file. Can we simplify all of them by calling __get_config() instead?

Reply:
it could be, let me check